<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WGShadeLineEditor</class>
 <widget class="QWidget" name="WGShadeLineEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>12</number>
     </property>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="sbRangeSaturation">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelRange">
       <property name="text">
        <string>Range:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="sbRangeHue">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="labelValue">
       <property name="text">
        <string>Value:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QDoubleSpinBox" name="sbOffsetValue">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labelHue">
       <property name="text">
        <string>Hue:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QDoubleSpinBox" name="sbOffsetSaturation">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QDoubleSpinBox" name="sbRangeValue">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="sbOffsetHue">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="labelSaturation">
       <property name="text">
        <string>Saturation:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelOffset">
       <property name="text">
        <string>Offset:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelDisplay">
       <property name="text">
        <string>Display:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbSlider">
       <property name="text">
        <string>S&amp;lider</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbColorPatches">
       <property name="text">
        <string>Color Pa&amp;tches</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbPatchCount">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>3</number>
       </property>
       <property name="value">
        <number>9</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelPreview">
     <property name="text">
      <string>Preview:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sbRangeHue</tabstop>
  <tabstop>sbRangeSaturation</tabstop>
  <tabstop>sbRangeValue</tabstop>
  <tabstop>sbOffsetHue</tabstop>
  <tabstop>sbOffsetSaturation</tabstop>
  <tabstop>sbOffsetValue</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
